supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -104,6 +104,42 @@ Write all titles and headings, including the titles of product documentation gui
 * _Configuring the node port service range_
 * _How to perform an unsupported conversion from a RHEL-derived Linux distribution to RHEL_
 
+[[commands-in-code-blocks]]
+== Commands in code blocks
+
+As a default, use bold formatting for commands in code blocks, to visually distinguish them from their lead-in sentences and example outputs.
+
+.Example AsciiDoc: Command and its output in a code block
+
+  Verify that the `libvirt` default network is active and configured to start automatically:
+  +
+  [subs="+quotes"]
+  ----
+  # *virsh net-list --all*
+  Name      State    Autostart   Persistent
+  --------------------------------------------
+  default   active   yes         yes
+  ----
+
+
+This renders as:
```

Comment:
Incomplete sentence. How about: `The previous AsciiDoc example outputs as follows`?

I'm suggesting this in the event that a new writer scans the RH SSG (to validate term usability) to check for instances of "This renders as" and they then apply it to customer docs.

Reply:
This exact wording was actually used in multiple other examples in the section, and I just copied it as a "template". Doesn't change the fact that it's an incomplete sentence, so I'll update the other occurrences, too.